[2025-04-04T11:23:03.381+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-04T11:23:03.400+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-04T11:22:57.628657+00:00 [queued]>
[2025-04-04T11:23:03.407+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-04T11:22:57.628657+00:00 [queued]>
[2025-04-04T11:23:03.408+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-04T11:23:03.418+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): meteo_operations.collect_meteo_data> on 2025-04-04 11:22:57.628657+00:00
[2025-04-04T11:23:03.425+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'meteo_operations.collect_meteo_data', 'manual__2025-04-04T11:22:57.628657+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/operators/testdag.py', '--cfg-path', '/tmp/tmpolr0rr0z']
[2025-04-04T11:23:03.428+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask meteo_operations.collect_meteo_data
[2025-04-04T11:23:03.429+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=204) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-04T11:23:03.430+0000] {standard_task_runner.py:72} INFO - Started process 222 to run task
[2025-04-04T11:23:03.477+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-04T11:22:57.628657+00:00 [running]> on host 0fee6e63c11b
[2025-04-04T11:23:03.577+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='meteo_operations.collect_meteo_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T11:22:57.628657+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-04T11:22:57.628657+00:00'
[2025-04-04T11:23:03.579+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-04T11:23:03.611+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 21.4901, 'longitude': 39.1862, 'timestamp': '2025-04-04 00:00:00', 'temperature': 25.473501205444336, 'humidity': 72.0, 'wind_speed': 8.121970176696777, 'pm2_5': 143.0, 'co': 201.0}
[2025-04-04T11:23:03.638+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-04T11:23:03.639+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=meteo_operations.collect_meteo_data, run_id=manual__2025-04-04T11:22:57.628657+00:00, execution_date=20250404T112257, start_date=20250404T112303, end_date=20250404T112303
[2025-04-04T11:23:03.685+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-04T11:23:03.711+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-04T11:23:03.713+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
